python/config/inactive-users.toml

```diff
@@ -0,0 +1,15 @@
+[github]
+organisation_name = "ministryofjustice"
+
+[activity_check]
+inactivity_months = 18
+
+[team.operations_engineering]
+github_team = "operations-engineering"
+slack_channel = "#operations-engineering-alerts"
+remove_from_team = true
+users_to_ignore = [ "cloud-platform-moj", "ScottSeaward" ]
```

Comment:
Is this an example file? Just thinking would you add all the bot accounts in this section?

Reply:
These are actual examples of accounts that have been flagged. In my opinion, there is an argument for both of these to be removed. No other bot accounts were flagged.